md-dao/src/main/java/com/wipro/ats/bdre/md/dao/HistoryDataImportDAO.java

```diff
@@ -41,8 +41,15 @@
     private static final Logger LOGGER = Logger.getLogger(HistoryDataImportDAO.class);
     @Autowired
     SessionFactory sessionFactory;
-
-    public List<com.wipro.ats.bdre.md.beans.table.Process> historyDataImport(IntermediateInfo intermediateInfo) throws Exception {
+    private static final String _IMPORT="_import";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00115)